api/v1alpha1/virtualmachineclass_types.go

```diff
@@ -21,6 +21,30 @@ type DynamicDirectPathIODevice struct {
 	CustomLabel string `json:"customLabel,omitempty"`
 }
 
+// InstanceStorage encapsulates information about instance storage.
+// StorageClass refers to the name of a StorageClass resource used to
+// provide the storage for the configured, instance storage volumes.
+// The value of this field has no relationship to or bearing on the field
+// virtualMachine.spec.storageClass.
+// Please note the referred StorageClass must be available in the same
+// namespace as the VirtualMachineClass that uses it for configuring
+// instance storage.
+type InstanceStorage struct {
+	// StorageClass describes the configuration of instance storage related
```

Comment:
Maybe...

```go
// StorageClass refers to the name of the StorageClass resource used to
// provide the storage for the configured instance storage volumes.
```

Reply:
I think @akutz suggestion is much clearer should be used for the field description instead of the putting it above in the struct description.